draft-packages/form/KaizenDraft/Form/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+
+import { InputSearch } from "../Primitives/InputSearch"
+import { FieldGroup, Label } from ".."
+export interface SearchFieldProps
+  extends Omit<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    "className" | "defaultValue"
+  > {
+  reversed?: boolean
+  loading?: boolean
+  secondary?: boolean
+  labelText?: React.ReactNode
+  onClear?: () => void
+}
+
+const SearchField: React.FunctionComponent<SearchFieldProps> = ({
+  id,
+  disabled = false,
+  placeholder,
+  reversed = false,
+  loading,
+  secondary,
+  value,
+  labelText,
+  onClear,
+  ...genericInputProps
+}) => {
+  const searchFieldId = `${id}-search-field-input`
+
+  return (
+    <FieldGroup inline={false}>
+      <Label
```

Comment:
This is rendering a blank `label` element for secondary variants. Probs just wanna put this behind a `!secondary` conditional.